src/relay/transforms/combine_parallel_batch_matmul.cc

```diff
@@ -68,6 +68,13 @@ class ParallelBatchMatmulCombiner : public ParallelOpCombiner {
     // shape[2] is the contraction axis and automatically consistent
     // if it were valid batch_matmul ops
 
+    // This pass only support the original NT format now
+    // TODO(jcf94): Add full support of layout format
+    if (attrs_a->transpose_a == true || attrs_a->transpose_b == false ||
+        attrs_b->transpose_a == true || attrs_b->transpose_b == false) {
+      return false;
```

Comment:
This is a required functionality so maybe throw a warning?

Reply:
Warning added.